Makefile

```diff
@@ -20,7 +20,8 @@ interrogate:
 
 lint:
 	pre-commit run markdownlint -a && \
-	pre-commit run ruff -a 
```

Comment:
Not sure there is a better place, but I figure I will open the discussion: Is `lint` the right place to put the large file check?

Reply:
Created one here: https://github.com/NVIDIA/modulus/issues/190

Reply:
I agree lint is an odd stage for this but there's not a better spot at this moment... updating stages breaks all repos so this will have to be a coordinated effort to update when ever that happens